packages/server/generate-subgraph-schema.ts

```
@@ -0,0 +1,18 @@
+import 'reflect-metadata';
+import { writeFileSync } from 'fs';
```

ESLint

[error] Prefer `node:fs` over `fs`.

(unicorn/prefer-node-protocol)